Makefile

```diff
@@ -71,6 +71,10 @@ fmt: ## Run go fmt against code
 vet: ## Run go vet against code
 	go vet ./...
 
+.PHONY: lint
+lint: golangci-lint ## Run golangci-lint against code
```

Comment:
[q] do we really need a separate lint goal considering that we have a precommit hook that will fix the code on each commit?

As an alternative - I'd suggest to [call](https://pre-commit.com/#usage) `pre-commit run --all-files` instead that will run all hooks on all the files. So if we add more hooks - all of them will be run here automatically

Reply:
I agree this is kind of redundant - will remove.